vcr/api/oidc4vci/v0/api.go

```diff
@@ -119,18 +115,46 @@ func (w Wrapper) Routes(router core.EchoRouter) {
 	}))
 }
 
+func (w Wrapper) getWallet(ctx context.Context, holderDIDAsString string) (holder.OIDCWallet, error) {
+	holderDID, err := w.validateDIDIsOwned(ctx, holderDIDAsString)
+	if err != nil {
+		return nil, err
+	}
+	return w.VCR.GetOIDCWallet(ctx, holderDID)
+}
+
+func (w Wrapper) getIssuer(ctx context.Context, issuerAsString string) (openid4vci.Issuer, error) {
```

Comment:
same as above, this is the IssuerHandler?

Reply:
Fixed